packages/trader/src/AppV2/Components/TradeParameters/Stake/stake.tsx

```diff
@@ -166,6 +188,14 @@ const Stake = observer(({ is_minimized }: TStakeProps) => {
         }
     }, [details, is_open, max_payout, max_stake, min_stake, first_contract_payout, second_contract_payout]);
 
+    React.useEffect(() => {
+        if (!is_focused) {
+            setShouldShowError(!!stake_error);
+        } else if (!amount) {
+            setShouldShowError(false);
+        }
+    }, [is_focused]);
```

Comment:
missing dependency - amount? or is that by purpose?

Reply:
also - my brain freezes when there are too many negative conditionals :D

what about:

```
React.useEffect(() => {
    if (is_focused) {
        if (amount) {
            setShouldShowError(false);
        }
    } else {
        setShouldShowError(!!stake_error);
    }
}, [is_focused]);
```

reason being is that positive conditionals are way easier to read and analyse,

Reply:
also stake_error dep might be needed?

Reply:
@wojciech-deriv I made changes to this useEffect as per your suggestions, as well as a new requirement from design